src/utils.table.ts

```
@@ -0,0 +1,77 @@
+	constructor(options: TableOptions) {
+		this.head = options.head;
+		this.colAligns = options.colAligns ?? Array.from({ length: this.head.length }, () => 'left');
+		this.style = options.style;
+	}
+
+	push(row: TableRow): void {
+		this.rows.push(row);
+	}
+
+	toString(): string {
+		const terminalWidth = process.stdout.columns || 120;
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)